backend/Dockerfile

```diff
@@ -29,12 +29,9 @@ RUN if [ "${ADD_PARSER}" = "1" ]; then python3 -m pip install --use-pep517 --no-
 ARG ADD_VECTORDB=0
 RUN if [ "${ADD_VECTORDB}" = "1" ]; then python3 -m pip install --use-pep517 --no-cache-dir -r /tmp/vectordb.requirements.txt; fi
 
-# Temporary addition until templates have been updated using build args as environment variables
-ARG ADD_RERANKER_SVC_URL=""
```

Comment:
Can we keep this, reranker is still required when deploying via templates, as there is no way to inject env vars at the moment.

Reply:
Simply setting it in env var in TrueFoundry spec should work?

Reply:
I'll revert this but this is not the correct solution
Any change requires a rebuild